nengo/builder/connection.py

```diff
@@ -298,6 +309,10 @@ def get_prepost_signal(is_pre):
 
     # Build learning rules
     if conn.learning_rule is not None:
+        if isinstance(conn.transform, Convolution):
```

Comment:
I'm less worried about this, but ultimately, it might be nice to have a way that users can define if their transform supports learning, and if so specify how the learning happens. I don't think this is high priority, though, so I'm happy to leave that as future work.

Reply:
I put a TODO marking this as something we should do in the future